### PR TITLE
OHARA-1442 Update backend image starting command in README.md

### PR DESCRIPTION
**Pull request by konekoya**

### CODE REVIEW

**Comment by chia7712** [comment]
麻煩加上住解說com.island.ohara.demo.Backend is deprecated and will be removed in 3.0. Also, please describe how to start ftp/mysql process.

**Comment by konekoya** [comment]
Good call. These should be included

**Comment by konekoya** [comment]
I think you mean "removed in v0.3" right?

**Comment by konekoya** [comment]
Updated. Would you mind to take a quick look @chia7712 :)

**Comment by konekoya** [comment]
Friendly ping to @chia7712, need your help on this :))

**Comment by konekoya** [comment]
A quick question for @vitojeng 

Sorry if you've told us about this before. But how do I label this PR? It's only updating README.md...

**Comment by konekoya** [comment]
> A quick question for @vitojeng
> 
> Sorry if you've told us about this before. But how do I label this PR? It's only updating README.md...

Ping @vitojeng :)

**Comment by konekoya** [comment]
> Updated. Would you mind to take a quick look @chia7712 :)

@chia7712 let's get this PR going :)

**Comment by chia7712** [comment]
It seems to me “infra” label is fine

File: README.md [hunk 1]
```diff
@@ -38,7 +42,19 @@ docker run --rm -p 12345:12345 oharastream/ohara:backend --configuratorPort 1234
...
+#### MySQL
+
+```
+docker run --rm -p 12345:12345 oharastream/ohara:backend mysql --configuratorPort 12345
+```
+
+#### FTP
+
+```
+docker run --rm -p 12345:12345 oharastream/ohara:backend ftp --configuratorPort 12345
```
Comment (chia7712) [inline README.md]: docker run —rm -p 10000-10010 :10000-10010 oharastream/ohara:backend com.island.ohara.integration.FtpServer —controlPort 10000 —dataPorts 10001-10011

File: README.md [hunk 2]
```diff
@@ -38,7 +42,19 @@ docker run --rm -p 12345:12345 oharastream/ohara:backend --configuratorPort 1234
 * ftpPort: bound by ftp server (default is random)
 * ttl: time to terminate backend-service (default is 365 days)
 
-The backend image is not in production release. Hence, there is no guarantees to backend image.
+The backend image is not included in production release. Hence, there is no guarantee to backend image.
+
+#### MySQL
+
+```
+docker run --rm -p 12345:12345 oharastream/ohara:backend mysql --configuratorPort 12345
```
Comment (chia7712) [inline README.md]: docker run —rm -p 20000:20000 oharastream/ohara:backend con.island.ohara.integration.Database —port 20000

Comment (konekoya) [reply README.md]: Addressed both comments

File: README.md [hunk 3]
```diff
@@ -38,7 +42,19 @@ docker run --rm -p 12345:12345 oharastream/ohara:backend --configuratorPort 1234
...
+#### MySQL
+
+```
+docker run —rm -p 20000:20000 oharastream/ohara:backend con.island.ohara.integration.Database —port 20000
+```
+
+#### FTP
+
+```
+docker run —rm -p 10000-10010 :10000-10010 oharastream/ohara:backend com.island.ohara.integration.FtpServer —controlPort 10000 —dataPorts 10001-10011
```
Comment (chia7712) [inline README.md]: 歹勢，這邊多了一個空格 10000-10010" ":10000-10010

File: README.md [hunk 4]
```diff
@@ -38,7 +42,19 @@ docker run --rm -p 12345:12345 oharastream/ohara:backend --configuratorPort 1234
 * ftpPort: bound by ftp server (default is random)
 * ttl: time to terminate backend-service (default is 365 days)
 
-The backend image is not in production release. Hence, there is no guarantees to backend image.
+The backend image is not included in production release. Hence, there is no guarantee to backend image.
+
+#### MySQL
+
+```
+docker run —rm -p 20000:20000 oharastream/ohara:backend con.island.ohara.integration.Database —port 20000
```
Comment (chia7712) [inline README.md]: --port
(我之前是用手機敲comment...好像格式有跑掉...)

File: README.md (hunk 3 above)
Comment (chia7712) [reply README.md]: --controlPort --dataPorts
(我之前是用手機敲comment...好像格式有跑掉...)

**Comment by konekoya** [comment]
@chia7712 Could take a look again. I've addressed these previous comments. Thanks :)

**Comment by konekoya** [comment]
Thanks for reviewing!

**Pull request merged**